### DISCUSSION TITLE
The link for designer visual editor is not working

### DISCUSSION THREAD

**Discussion by manjindersinghsarkaria** [primefaces/community#1439] [PrimeVue]
The https://designer.primevue.org link is broken

**Comment by ferguson03** [comment]
i reported this yesterday on a youtube channel video covering the designer.  It is 404ing

**Comment by autonomobil** [comment]
yep, not nice. Linked in a couple of places

**Comment by jonasjancarik** [comment]
It seems to be here https://www.primefaces.org/designer-jsf/. Try different base themes to see more options

**Comment by jogilham** [comment]
This seems to not be working either now... any ideas where it might have got to?
